src/python/deepgnn/graph_engine/snark/tests/convert_test.py

```diff
@@ -26,7 +35,12 @@ def triangle_graph_json(folder):
             "node_weight": 1,
             "neighbor": {"0": {"0": 0.5}, "1": {}},
             "uint64_feature": {},
-            "float_feature": {"0": [0, 1], "1": [-0.01, -0.02]},
+            "float_feature": [[0, 1], [-0.01, -0.02]]
```

Comment:
Please, [update documentation for proposed changes](https://github.com/microsoft/DeepGNN/blob/main/docs/advanced/data_spec.md#graph-data)